static/app/views/settings/account/accountClose.tsx

```diff
@@ -147,16 +150,19 @@ function AccountClose() {
       </Alert>
 
       <Panel>
-        <PanelHeader>{t('Remove the following organizations')}</PanelHeader>
+        <PanelHeader>{t('Delete the following organizations')}</PanelHeader>
         <PanelBody>
           <PanelAlert type="info">
+            {tct('[strong:ORGANIZATIONS WITH CHECKED BOXES WILL BE DELETED]', {
```

Comment:
i see that we do this in a couple places, but I'm not the biggest fan. `tct` is best for nesting components, which we aren't really doing here. This could just be a `t` wrapped in a `strong`